README.md

```diff
@@ -151,7 +153,7 @@ each individual UI element is in
 
 ## Testing
 
-### Playwright tests
+### Playwright Tests
```

Comment:
This PR seems to also contain changes to the textual content. I prefer that these are made in a separate PR so that we only talk about which files don't get prettified here.

Reply:
The changes are minimal so I don't see the point of opening more PRs. Let's keep it this way.